src/test/java/Pages/LoginPage.java

```diff
@@ -0,0 +1,38 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class LoginPage extends BasePage{
+    By submitbutton = By.cssSelector("button[type='submit']");
+    By emaillocator = By.cssSelector("input[type='email']");
+    By passwordlocator = By.cssSelector("input[type='password']");
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");
```